docs/brand-guide.rst

```diff
@@ -0,0 +1,84 @@
+####################
+Solidity Brand Guide
+####################
+
+This brand guide features information on Solidity's brand policy and
+logo usage guidelines.
+
+The Solidity Brand
+==================
+
+The Solidity programming language is an open-source, community project
+governed by a core team. The core team is sponsored by the `Ethereum
+Foundation <https://ethereum.foundation/>`_.
+
+This document aims to provide information about how to best use the
+Solidity brand name and logo.
+
+We encourage you to read this document carefully before using the
+brand name or the logo. Your cooperation is highly appreciated!
+
+Solidity Brand Name
+===================
+
+"Solidity" should be used to refer to the Solidity programming language
+solely.
+
+Please do not use "Solidity":
+
+- To refer to any other programming language.
+
+- In a way that is misleading or may imply association of unrelated
+modules, tools, documentation, or other resources with the Solidity
+programming language.
+
+- In ways that confuse the community as to whether the Solidity
+programming language is open-source and free to use.
+
+Solidity Logo License
+=====================
+
+.. image:: https://i.creativecommons.org/l/by/4.0/88x31.png
+  :width: 88
+  :alt: Creative Commons License
+
+The Solidity logo is distributed and licensed under a `Creative Commons
+Attribution 4.0 International License <http://creativecommons.org/licenses/by/4.0/>`_.
+
+This is the most permissive Creative Commons license and allows reuse
+and modifications for any purpose.
+
+You are free to:
+
+- **Share** — Copy and redistribute the material in any medium or format.
+
+- **Adapt** — Remix, transform, and build upon the material for any
+purpose, even commercially.
+
+Under the following terms:
+
+- **Attribution** — You must give appropriate credit, provide a link to
+the license, and indicate if changes were made. You may do so in any
+reasonable manner, but not in any way that suggests the the Solidity
+core team endorses you or your use.
+
+When using the Solidity logo, please respect the Solidity logo guidelines.
+
+Solidity Logo Guidelines
+========================
+
+.. literalinclude:: logo.svg
```

Comment:
It's also complaining about this line, but I'm not sure what the intention here is

Reply:
We should just use `.. image:: logo.svg` instead.

Reply:
Width `width: 128`, 160, or 256 it looks nice.